hackathon_site/registration/jinja2/registration/emails/rejected_email_body.html

```diff
@@ -0,0 +1,7 @@
+<p>Hello {{ user.first_name|striptags }},</p>
+
+<p>Thank you for your interest in {{hackathon_name}} 2020. While we appreciate your time and energy, we are not able to offer you a spot in the event this year. We hope that this decision does not prevent you from applying next year! Every year we look to increase the number of attendees. If you have any questions or concerns, feel free to contact us!</p>
```

Comment:
I wouldn't put 2020 in here, as it's something we would need to remember to change in the future. If people want it in their templates, they could always put it as part of the `hackathon_name` variable.